modules/ROOT/pages/appendix/version-compatibility.adoc

```diff
@@ -12,62 +12,61 @@ Neo4j Ops Manager follows link:https://semver.org/[semantic versioning] for its
 
 == Compatibility matrix
 
+[NOTE]
+====
+The NOM agent version should always match the current NOM server version. Other configurations may work, but are unsupported.
```

Comment:
```suggestion
The NOM agent version should always match the current NOM server version.
Other configurations may work, but are unsupported.
```
Docs Team wish to have every sentence on a new line.

Reply:
In notes, you mean?